MatrixSDK/Crypto/Verification/MXKeyVerificationManager.m

```diff
@@ -609,7 +609,10 @@ - (MXHTTPOperation*)sendToOtherInRequest:(MXKeyVerificationRequest*)request
             break;
             
         case MXKeyVerificationTransportToDevice:
-            operation = [self sendToDevice:request.otherUser deviceId:request.otherDevice eventType:eventType content:content success:success failure:failure];
+            if (request.otherDevice)
```

Comment:
I am not confortable with this patch. I am even happy it crashes.
If we cannot communicate with the other peer, we have a serious problem. The bug seems in another place or at another level.

Can you describe the issue?